.github/workflows/make-test.yml

```diff
@@ -47,6 +51,12 @@ jobs:
           ocaml-compiler: ${{ matrix.cfg.ocaml-version }}
           dune-cache: true
 
+      - run: |
+          opam pin add dune 3.5.0
+          opam pin add menhir 20220210
```

Comment:
Increasing `memhir` minimal version to "202401715" should fix the run failure. However, I'd like to have your opinion on comparing this PR with PR #1423.

Reply:
We may need a more recent version of `dune` to deal with  the more recent version of `menhir`.

Reply:
You mean that the need for recent versions originates in  one of our dune file?